src/new-client/src/App/LiveRates/Tile/Header/TileHeader.tsx

```diff
@@ -64,40 +64,50 @@ export const Header: React.FC = () => {
   } = useTileContext()
   const date = useDate(symbol)
   const canTearOut = supportsTearOut
+  const onClick = () => {
+    tearOut(symbol, !isTornOut)
+  }
+  return (
+    <HeaderComponent
+      canTearOut={canTearOut}
+      date={date}
+      base={base}
+      terms={terms}
+      isTornOut={isTornOut}
+      onClick={onClick}
+    />
+  )
+}
 
+export const HeaderComponent: React.FC<{
+  canTearOut?: boolean
+  date: any
+  base: any
+  terms: any
+  isTornOut?: boolean
+  onClick?: () => void
+  tearOutMock?: boolean
+}> = ({
+  canTearOut,
+  base,
+  terms,
+  date,
+  isTornOut,
+  onClick,
+  tearOutMock = false,
+}) => {
   return (
-    <HeaderWrapper>
+    <HeaderWrapper tearOutMock={tearOutMock}>
       <TileSymbol data-qa="tile-header__tile-symbol">
         {base}/{terms}
       </TileSymbol>
       <DeliveryDate data-qa="tile-header__delivery-date">{date}</DeliveryDate>
-      {canTearOut && (
-        <HeaderAction
-          onClick={() => {
-            tearOut(symbol, !isTornOut)
-          }}
-        >
+      {canTearOut && !tearOutMock && (
+        <HeaderAction onClick={onClick ? onClick : () => {}}>
           {isTornOut ? <PopInIcon /> : <PopOutIcon />}
         </HeaderAction>
       )}
-    </HeaderWrapper>
-  )
-}
-
-export const HeaderComponent: React.FC<{
-  supportsTearOut: boolean
-  currencyPair: any
-}> = ({ currencyPair, supportsTearOut }) => {
-  const dateVal = `SPT (${format(new Date("08/04"), "dd MMM").toUpperCase()})`
-  return (
-    <HeaderWrapper supportsTearOut={supportsTearOut}>
-      <TileSymbol data-qa="tile-header__tile-symbol">
-        {currencyPair.base}/{currencyPair.terms}
-      </TileSymbol>
-      <DeliveryDate data-qa="tile-header__delivery-date">
-        {dateVal}
-      </DeliveryDate>
-      {supportsTearOut && (
+      {tearOutMock && (
```

Comment:
Can we remove this duplication so we only have 1 set of icons in the render function? 
You should be able to achieve the same styling if you pass `shouldMoveDate={true}` when `hover` is true in the `TileMockComponent`. That will add the styles to the `DeliveryDate` meaning the absolute positioning should be correct for the pop out icon